src/it/strict-bundled-artifacts-extra/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>5.18</version>
+    <relativePath />
+  </parent>
+
+  <groupId>org.jenkins-ci.tools.hpi.its</groupId>
+  <artifactId>strict-bundled-artifacts-extra</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>hpi</packaging>
+
+  <properties>
+    <jenkins.version>2.516.1</jenkins.version>
+    <hpi-plugin.version>@project.version@</hpi-plugin.version>
+    <spotless.check.skip>false</spotless.check.skip>
+    <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+    <!-- slf4j-api is listed here but is not actually bundled -->
```

Comment:
OK, though it would seem more intuitive to me to test the case that you _forgot_ the `exclusion` and were bundling `slf4j-api` accidentally.

Reply:
I do test that case as well with Jackson dependencies in `strict-bundle-artifacts-missing`. `strict-bundled-artifacts-extra` is to make sure that `<hpi.bundledArtifacts>` isn't allowed to have unnecessary entries.